### DISCUSSION TITLE
Hiding Data Edit tab

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#1787] [Q&A] [answered]
@MoBosse dou you know how i hide this Data Edit tab?

site repo: https://github.com/christoph-LSN/MT_Site
data repo: https://github.com/christoph-LSN/IM_Data

![image](https://user-images.githubusercontent.com/94443931/186390422-af10f163-2baa-4676-970e-ec073235d772.png)

**Comment by MoBosse** [comment, marked as answer]
This tab should not be visible on your production site.
If you want to get rid of it on your staging site as well, simply overwrite [this file](https://github.com/open-sdg/open-sdg/blob/2.1.0-dev/_includes/components/indicator/metadata-tabs.html#L29-L33) and delete lines 29 to 33.

**Reply by Mazup** [reply]
Hi, I tried out these solution. So I created a new file "metadata-tabs.html" into the "_includes/components/indicator" folder. But my changes in this file hasn't any impact. What's wrong? Is there an additional setting to make? Thx says Matthias

**Comment by christoph-LSN** [comment]
@MoBosse Ah ok, i haven't thought there is a difference between my prodiction an my staging site!
Thanks a lot!

**Comment by SandraEliz** [comment]
Hello.

I don't have that file, what should I do to remove the edit tab?

Thank you

**Reply by sarah-beasley** [reply]
Hi,

An alternative could be the following:

In your site_config.yml file https://github.com/ods-mexico1-gmail/site/blob/cbb5efd918687c7d496d24c9cdeddfe2e1210508/_data/site_config.yml#L116, under the indicator_config_form indicator_data_form and indicator_metadata_form headings - change enabled: true, to enabled: false. This should remove them.

Hope this helps!

Sarah

**Comment by SandraEliz** [comment]
Hi

When changing those properties I get the following error
![image](https://user-images.githubusercontent.com/117297044/201124615-9f4bb08e-a234-4354-aaa9-bede086c9cde.png)
